minutes/minutes-2017-11-14.md

```diff
@@ -0,0 +1,110 @@
+     - These two above instruments may prevent some amount of money moving from NumFOCUS to CI.
+     - Tracy is thinking of leaving Data Carpentry as an Open Project at NumFOCUS. I have advised that optically this does not look good when we’re trying to merge and close projects. I have recommended that we ask NumFOCUS to allow us to create a new project that will have a clean set of books to run these, and only these two projects through.
+- What’s in a name….aka THE BOARD OF DIRECTORS ain’t gonna fly.
+    - CI calls project leadership teams Advisory Committees
+    - Ethan White suggests we ask if we can form an Executive Committee in our structure, so people get THAT sweet-as title.
+    - We want something that gives a bit of cache to our committee members, but we also want to be compliant. 
+- State of memberships:
+    - There are 56 Current Members - 4 out for signatures - 8 very close to onboarding and 12 in the wings with terms being finalized. For reference: total payroll across the carpentries is at about $700,000/year
+    - Of the active memberships, we have 30 new members (in the trailing 12-months), so we’ve doubled in revenue since this time last year. With 20 other memberships close to signing, we’re going to quite a clip!
+    - Instructors: We have commitments to train 698 instructors in our 56 active agreements. 
+
+ 
+#### Minutes
+- The fiscal committee will approve the dissolving of the existing steering committees and with the approval of the community. 
+- The linked document will be the agreement to transition from NumFocus to Community Initiatives as our fiscal sponsor. We can send an email vote to approve. CI’s policies and procedures for projects 
+- We will need to start moving money. CI will receive a 10% “tax” on all money transferred.
+- Jonah is still signing some partnerships that are going through NumFocus because of their organization's strength, rather than postponing it until January. 
+- All ~50 current members will have to sign a form saying they agree that they need to move finances from NumFocus to CI. Most organizations understand that this is in our and their best interest. 
+- **Action item for the merger committee/future steering committee**: email to partners saying old organizations and old fiscal sponsorship dissolved and announcing the new organizations and sponsorship will move forward.
+- Data Carpentry just got a grant from the Moore Foundation, but the money goes to NumFocus. Jonah and Tracy both have ideas for how this money can stay with NumFocus but be used by the Carpentries, which will be sponsored by CI. This is a problem that we need to find a good solution because we need to be able to pay staff with this money and have them be paid as staff via CI rather than as self-employed. 
```

Comment:
Are you sure this is Moore? https://github.com/swcarpentry/board/pull/246/files#diff-4ecc37558e5c39d6f014e5101659a41eR67 says Sloan.